src/App.js

```diff
@@ -1,23 +1,64 @@
-import logo from './logo.svg';
 import './App.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import Image from './Image';
+import Form from './Form';
+import title from './title.png';
 
 function App() {
+
+  const [pokemon, setPokemon] = useState({
+    name: "",
+    img: "",
+  });
+
+  const [num, setNum] = useState(123);
+
+  const [userChoice, setUserChoice] = useState('');
+
+  const [counter, setCounter] = useState(0);
+
+  const randomNumber = (num) => {
+    return Math.floor(Math.random() * num);
+  }
+
+  useEffect (() => {
+    setNum(randomNumber(252));
```

Comment:
is 252 the length of the resonse array?  if so, maybe set randomNumber = response.length ( or whatever the path is)